### PR TITLE
Example to run DenseNet with IPEX

### PR DESCRIPTION
**Pull request by swanandmhalagi**
We have added an example to run with the Python (PyTorch + IPEX) backend.

### CODE REVIEW

**Review by Antonyvance — CHANGES_REQUESTED** [review]
Suggesting following changes:
- Change folder name "serving_densnet_with_ipex_triton" (I will change folder name for Bert-large)

**Review by Antonyvance — CHANGES_REQUESTED** [review]
Please add link to "Samples list"

File: serving_with_ipex_triton/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- DenseNet121        - PyTorch+IPEX [DenseNet121](https://pytorch.org/hub/pytorch_vision_densenet/ "DenseNet121")
+
+## Possible run scenarios
+AI Inference samples allow user to run inference on localhost or on remote Triton Server Host. 
+By default config.properties is filled with localhost run option. 
+
+### Execution on localhost
+To build, start Docker containers, run tests, stop and do cleanup on localhost execute scripts in following order:
+
+`$ docker build -t tritonserver_custom -f Dockerfile.compose .`  - builds Docker image for Triton Server
```
Comment (cepera) [inline serving_with_ipex_triton/README.md]: looks like the name is actually different 
```suggestion
`$ docker build -t tritonserver_custom -f Dockerfile.ipex .`  - builds Docker image for Triton Server
```

File: serving_with_ipex_triton/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Support
+Please submit your questions, feature requests, and bug reports on the [GitHub issues page](https://github.com/intel/intel-ai-inference-samples/issues).
+
+## License 
+AI Inference samples project is licensed under Apache License Version 2.0. Refer to the [LICENSE](../LICENSE) file for the full license text and copyright notice.
+
+This distribution includes third party software governed by separate license terms.
+
+3-clause BSD license:
+- [model.py](./model_utils/bert_common/1/model.py) -  for PyTorch (IPEX)
```
Comment (cepera) [inline serving_with_ipex_triton/README.md]: this path's also incorrect
 ```suggestion
- [model.py](./model_repository/densenet/1/model.py) -  for PyTorch (IPEX)
```

File: serving_with_ipex_triton/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+This third party software, even if included with the distribution of the Intel software, may be governed by separate license terms, including without limitation, third party license terms, other Intel software license terms, and open source software license terms. These separate license terms govern your use of the third party programs as set forth in the [THIRD-PARTY-PROGRAMS](./THIRD-PARTY-PROGRAMS) file.
```
Comment (cepera) [inline serving_with_ipex_triton/README.md]: the `THIRD-PARTY-PROGRAMS` file doesn't exist here

File: serving_with_ipex_triton/client_imagenet.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    img = Image.open(img_path)
+    preprocess = transforms.Compose([
+        transforms.Resize(224),
+        transforms.CenterCrop(224),
+        transforms.ToTensor(),
+        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+    ])
+    return preprocess(img).numpy()
+
+directory = '/home/ubuntu/ImageNet/imagenet_images'
```
Comment (cepera) [inline serving_with_ipex_triton/client_imagenet.py]: please make the path to the dataset a required parameter.

File: serving_with_ipex_triton/start.sh [hunk 5]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+print_help() {
```
Comment (cepera) [inline serving_with_ipex_triton/start.sh]: help functions seems to be obsolete

File: serving_with_ipex_triton/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+## Possible run scenarios
+AI Inference samples allow user to run inference on localhost or on remote Triton Server Host. 
+By default config.properties is filled with localhost run option. 
+
+### Execution on localhost
+To build, start Docker containers, run tests, stop and do cleanup on localhost execute scripts in following order:
+
+`$ docker build -t tritonserver_custom -f Dockerfile.compose .`  - builds Docker image for Triton Server
+
+`$ bash start.sh`  - runs Docker containers for Triton Server Client and Host for model specified in config.properties
```
Comment (cepera) [inline serving_with_ipex_triton/README.md]: it looks like only host server is being run in `start.sh`

Comment (cepera) [reply serving_with_ipex_triton/README.md]: `config.properties` doesn't exist and not used

**Review by cepera — CHANGES_REQUESTED** [review]

File: serving_with_ipex_triton/client_imagenet.py (hunk 4 above)
Comment (swanandmhalagi) [reply serving_with_ipex_triton/client_imagenet.py]: Added mandatory parameter for dataset

File: serving_with_ipex_triton/start.sh (hunk 5 above)
Comment (swanandmhalagi) [reply serving_with_ipex_triton/start.sh]: Edited the help function for this usecase

File: serving_with_ipex_triton/README.md (hunk 6 above)
Comment (swanandmhalagi) [reply serving_with_ipex_triton/README.md]: Updated!

**Pull request merged**